### PR TITLE
New version number display, better fullscreen and self-hiding disclaimer

### PR DESCRIPTION
**Pull request by daalta**
### Purpose
Unity_2020-01-14_18-56-58

* The alpha disclaimer introduced in https://github.com/unitystation/unitystation/pull/2587 now self-hides when entering a game.
  * It hides if any scene is loaded that is not the lobby. The disclaimer is shown again when opening the escape menu.
* The build number text was changed and is now always displayed in the bottom-left of the screen.
  * This should be useful when looking at screenshots. We'll be able to tell what version of Unitystation is shown in feedback, community posts, bug reports etc.
* Added a new fullscreen option to the options menu.
  * Entering fullscreen sets the game resolution to the screen's native resolution. 
  * Exiting fullscreen defaults to a 1280 * 720 resolution.
  * Fullscreen- and resolution settings are automatically stored by Unity. Exiting the game *should* restore the previous settings.
  * This makes https://github.com/unitystation/unitystation/issues/2575 less of an issue. ALT+ENTER still sets the wrong resolution, but using the new menu option works file.
    * We could (should?) disable ALT+ENTER to enter fullscreen. However, this would mean that players cannot enter or exit fullscreen without joining a game.

### Notes

Pressing escape no longer exits fullscreen. (See https://github.com/unitystation/unitystation/issues/2575#issuecomment-574277144) I am not sure why that functionality was there, but looks like I did not break anything? 🤞
(Update: This was code leftover from when the game was always fullscreen)

### Please make sure you have followed the self checks below before submitting a PR:

- [x] Code is sufficiently commented
- [x] Code is indented with tabs and not spaces
- [x] The PR does not include any unnecessary .meta, .prefab or <b>.unity (scene) changes</b>
- [x] The PR does not bring up any new compile errors
- [x] The PR has been tested in editor

### CODE REVIEW

**Comment by daalta** [comment]
Doobly has some upcoming UIManager changes, so don't merge this until his changes are in. Then I'll resolve any conflicts.

**Comment by daalta** [comment]
Finally done with merging develop into my branch, this is ready for review. Had some issues - probably due to removed, renamed or conflicting files. We'll never knoooow 🌬

File: UnityProject/Assets/Scripts/UI/IngameMenu/GUI_IngameMenu.cs [hunk 1]
```diff
@@ -36,6 +36,19 @@ void Awake()
 		}
 	}
 
+	void OnEnable()
+	{
+		SceneManager.sceneLoaded += OnSceneLoaded;
```
Comment (Jibbajabbafic) [inline UnityProject/Assets/Scripts/UI/IngameMenu/GUI_IngameMenu.cs:41]: Make sure you add a corresponding OnDisable() to unregister this method or there could be issues in the future

**Review by Jibbajabbafic — CHANGES_REQUESTED** [review]
Everything else seems good, thanks!

**Comment by DooblyNoobly** [comment]
Looks really good. Just fix up the points noted by jibby and we'll merge it

**Comment by DooblyNoobly** [comment]
actually I'll fix this up in my pr. Merging now

**Comment by daalta** [comment]
Thanks @Jibbajabbafic and @DooblyNoobly doobly, appreciate it!

Fullscreen could still be a little more robust, but it should do for now.

**Pull request merged**